### PR TITLE
PORT-10679 Extend deployments docs with Gitlab & Azure DevOps

### PR DESCRIPTION
**Pull request by kodjomiles**
# Description
Extending the deployment strategies to cover other git providers (Azure and Gitlab)

## Updated docs pages

Please also include the path for the updated docs

- Quickstart (`/`)
- Blueprint (`/platform-overview/port-components/blueprint`)
- Dora Metrics Guide (`/guides/all/setup-dora-metrics`)

### CODE REVIEW

**Review by Matanga1-2 — APPROVED** [review]
Looks good, make sure to add @hadar-co as reviewer

**Comment by kodjomiles** [comment]
@Matanga1-2 I will but I want to update the gitlab docs and the azure devops docs with the leadTime for pr first before this is merged.. else we will find some misleading information about the leadTime ... it's still a calculation property on the gitlab docs and is not mentioned anywhere on azure. I will open a task for it shortly and implement them ASAP then the review can continue

File: docs/guides/all/setup-dora-metrics.md [hunk 1]
```diff
@@ -21,9 +21,9 @@ This guide will cover the four key metrics: **deployment frequency**, **lead tim
...
 - Complete the [Port onboarding process](https://docs.getport.io/quickstart).
-- While this guide demonstrates an implementation using GitHub, other Git providers can be used as well.
-- Access to a GitHub repository that is connected to Port via the onboarding process.
-- Optional for advanced strategies: If you're using workflows or pipelines, ensure they are configured for deployment tracking by following the relevant setup guides, such as CI/CD or GitHub Actions.
+- Access to a repository (GitHub, GitLab, or Azure Repos) that is connected to Port via the onboarding process.
+- While this guide demonstrates implementations using **GitHub**, **GitLab**, and **Azure Repos**, other Git providers can be used as well.
+- Optional for advanced strategies: If you're using workflows or pipelines, ensure they are configured for deployment tracking by following the relevant setup guides, such as CI/CD integrations or your platform-specific tools.
 
 
 ## Tracking Deployment
```
Comment (hadar-co) [inline docs/guides/all/setup-dora-metrics.md:29]:
```suggestion
## Tracking deployments
```

**Review by hadar-co — CHANGES_REQUESTED** [review]

File: docs/guides/all/setup-dora-metrics.md [hunk 2]
```diff
@@ -126,55 +131,166 @@ By following these steps, you can paste and manage the JSON schema required to t
...
+  {label: "GitLab", value: "gitlab-pr-merge"},
+  {label: "Azure Repos", value: "azure-repos-pr-merge"}
+  ]}>
 
-- When a PR is merged, a **deployment entity** is created in Port to represent the deployment that took place.
-- The **lead time** for that PR is calculated and added to the deployment as part of the blueprint.
+  <TabItem value="github-pr-merge" label="GitHub">
 
-Here’s how you can implement this:
+  Below is how the deployment tracking can be done in GitHub using the PR merge strategy:
```
[the remark this answers is not in the document]
Comment (hadar-co) [reply docs/guides/all/setup-dora-metrics.md:157]: @kodjomiles please change this phrasing in all instances, there quite a few

**Review by PeyGis — APPROVED** [review]

**Review by hadar-co — APPROVED** [review]

**Pull request merged**